core/src/services/compfs/mod.rs

```diff
@@ -14,3 +14,97 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
+
```

Comment:
We won't place anything in `mod.rs` expect `mod` and `use`. How about moving them to `core.rs`?

Reply:
sure